app/controllers/concerns/errorable.rb

```diff
@@ -4,6 +4,7 @@ module Errorable
   included do
     rescue_from Errors::FormatError, ActionController::UnknownFormat, with: :handle_format_error
     rescue_from Errors::AuthenticationError, with: :handle_authentication_error
+    rescue_from Errors::RequestError, with: :handle_request_error
     rescue_from Errors::Connectors::ConnectorError, with: :handle_connector_error
     rescue_from Errors::Connectors::ServiceError, with: :handle_service_error
```

Comment:
`rescue_from` accepts  a block as well. So, you could do
```ruby
rescue_from(Errors::Connectors::ServiceError) do |ex|
  render_error :service_unavailable, ex.message
end
```
and save a few `def ... end`s.